src/probnum/quad/_bayesquad.py

```diff
@@ -0,0 +1,68 @@
+"""
+Bayesian Quadrature.
+
+This module provides routines to integrate functions through Bayesian quadrature,
+meaning a model over the integrand is constructed in order to actively select evaluation
+points of the integrand to estimate the value of the integral. Bayesian quadrature
+methods return a random variable with a distribution, specifying the belief about the
+true value of the integral.
+"""
+
+from .bq_methods._bayesian_quadrature import (
+    VanillaBayesianQuadrature,
+    WSABIBayesianQuadrature,
+)
+
+
+def bayesquad(fun, fun0, domain, nevals=None, method="vanilla", **kwargs):
```

Comment:
There should also be an argument `integration_measure`, but we can add it alongside the integration measures at a later stage.